relayer/Cargo.toml

```diff
@@ -0,0 +1,43 @@
+[package]
+name = "fuel-relayer"
```

Comment:
Shouldn't this be a separate branch / PR?

Reply:
removed